src/sentry/tasks/app_store_connect.py

```diff
@@ -80,3 +81,14 @@ def create_difs_from_dsyms_zip(dsyms_zip: str, project: Project) -> None:
         created = debugfile.create_files_from_dif_zip(fp, project, accept_unknown=True)
         for proj_debug_file in created:
             logger.debug("Created %r for project %s", proj_debug_file, project.id)
+
+
+@instrumented_task(
+    name="sentry.tasks.app_store_connect.refresh_all_builds", queue="appstoreconnect"
+)
+def refresh_all_builds() -> None:
+    # TODO: well, we want to refresh *ALL* the projects that have an appstore
+    # connect symbol source, so how can I efficiently enumerate those and page
+    # through them without completely overloading the task system?
+    # inner_dsym_download(project_id=project_id)
```

Comment:
You can do a horrible hack: look at the `AppConnectBuild` table/model and get all projects which have appstore connect configured from there.
It relies on each project getting at least one entry on first successful configuration, which already is broken tough since someone might set up the a projects when there are not any builds yet.  So presumably this hack is of no use.

Reply:
another thing that's not too bad to iterate: `ProjectOption.objects.filter(key="sentry:symbol_sources")`